hw/ip/rv_timer/dv/rv_timer_sim_cfg.hjson

```diff
@@ -0,0 +1,68 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the IP under test.
+  name: rv_timer
+
+  // Top level dut name (sv module).
+  dut: rv_timer
+
+  // Top level testbench name (sv module).
+  tb: tb
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:rv_timer_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/rv_timer/data/rv_timer_testplan.hjson"
+
+  // RAL spec - used to generate the RAL model.
+  ral_spec: "{proj_root}/hw/ip/rv_timer/data/rv_timer.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: [// Project wide common sim cfg file
+                "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
+                // Common CIP test lists
+                "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
+                "{proj_root}/hw/dv/data/tests/intr_test.hjson",
+                "{proj_root}/hw/dv/data/tests/tl_access_tests.hjson",
+                "{proj_root}/hw/dv/data/tests/stress_tests.hjson"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  // Default UVM test and seq class name.
+  uvm_test: rv_timer_base_test
+  uvm_test_seq: rv_timer_base_vseq
+
+  // List of test specifications.
+  tests: [
+    {
+      name: rv_timer_sanity
+      uvm_test_seq: rv_timer_sanity_vseq
+    }
+
+    {
+      name: rv_timer_disabled
+      uvm_test_seq: rv_timer_disabled_vseq
+    }
+
+    {
+      name: rv_timer_cfg_update_on_fly
+      uvm_test_seq: rv_timer_cfg_update_on_fly_vseq
+      run_opts: ["+zero_delays=1", "+test_timeout_ns=10_000_000_000"]
+    }
+
+    {
+      name: rv_timer_random_reset
+      uvm_test_seq: rv_timer_random_reset_vseq
+    }
+
+    {
+      name: rv_timer_stress_all
+      uvm_test_seq: rv_timer_stress_all_vseq
+      run_opts: ["+test_timeout_ns=10_000_000_000"]
+    }
+  ]
```

Comment:
Are these two tests included in the common tests already? I think we can remove them here.

Reply:
Good catch, it looks like the *_stress_all test is included in stress_tests.hjson, but the rv_timer_random_reset test is not included in the imported sim_cfg files.